### PR TITLE
Empty lines cherry

### PR DESCRIPTION
**Pull request by danka74**
TLDR; skipping empty lines in RF2 files
Had a trailing CRLF in a RF2 file and wasted some precious time trying to identify the issue :(. I did not find any statement in the RF2 specification saying that empty lines are prohibited (perhaps they should be?) and thus skipping them seems appropriate. Added a INFO log when lines are skipped so empty lines should not pass unnoticed. The header is always the first line though, no initial empty lines allowed.
Also, added a check that each line of the RF2 file contains the same number of columns as the header, and found that one of the test files had spaces in place of a tab in the header (sct2_sRefset_OWLAxiomXExtensionSnapshot_XX_20170231.txt)!

### CODE REVIEW

**Review by kaicode — APPROVED** [review]
Thanks @danka74 this looks good. I'll run an interactive rebase to clean up the commit after I accept this and add the new version to Snowstorm.
